dist/tools/buildsystem_sanity_check/check.sh

```diff
@@ -16,43 +16,29 @@
 : "${RIOTBASE:="$(cd "$(dirname "$0")/../../../" || exit; pwd)"}"
 
 : "${RIOTTOOLS:=${RIOTBASE}/dist/tools}"
+# shellcheck disable=SC1091 (shellcheck is not run with -x in the CI)
```

Comment:
Ah wait, shell check is confused by the comment after apparantly:

> Invalid key=value pair? Ignoring the rest of this directive starting here. [SC1125]

Reply:
OK, I moved the non-semantic comment one line above. Should be good now :)